src/components/Dashboard/TodoList.vue

```diff
@@ -1690,11 +1702,6 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
     const restoration = filing.restoration
 
     if (header && restoration) {
-      // don't allow resuming a draft if not in good standing
-      if (this.isStatusDraft(header) && !this.isGoodStanding) {
-        task.enabled = false
-      }
```

Comment:
Don't prevent staff from resuming a draft restoration.

Note that regular users are prevented from doing any actions on a restoration on line 315 above.

Staff view:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/5e094e82-e0d8-41ef-b711-88103b6482d3)

Regular user view:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/ef96695d-2fcd-4cf5-b47a-49a806557d85)